### DISCUSSION TITLE
Does anyone know how to enable walk away lock on Koleos 2014?

### DISCUSSION THREAD

**Discussion by yanzhengdao** [cedricp/ddt4all#1269] [Q&A]
I've seen the function activated on a megane 2013 and other older cars at a dealer shop. But they refuse to help me to active it for me. Can I enable this feature with ddt4all?

**Comment by AndyIreland** [comment]
you might not have all the required extra hardware for it fitted to your particular car like aerials and needed ecu's or you might have an earlier version ecu and the setting not actually in the ecu where you can enable it
